docs/products/alloydb-omni/access-ai-models.md

```diff
@@ -0,0 +1,143 @@
+---
+title: Access and use AI models with Aiven for AlloyDB Omni®
+sidebar_label: Access AI models
+---
+
+import ConsoleLabel from "@site/src/components/ConsoleIcons";
+
+Enable and use AI models in Aiven for AlloyDB Omni® to build and deploy generative AI
+applications directly on your operational data.
+
+Aiven for AlloyDB Omni® allows you to integrate and use AI models such as OpenAI, Hugging
+Face, or Gemini.
+
+## Prerequisites
+
+- Aiven for AlloyDB Omni® service running
+- Access to the [Aiven Console](https://console.aiven.io)
+- [`psql` CLI client installed](https://www.postgresql.org/download/)
+- [`gcloud` CLI client installed](https://cloud.google.com/sdk/docs/install)
+- [Google service account credentials uploaded into Aiven for AlloyDB Omni](/docs/products/alloydb-omni/manage-credentials)
+
+## Use OpenAI
+
+Use the model by running:
```

Comment:
OpenAI model needs to be declared as well. I used this myself:

CALL  google_ml.create_model(
    model_id => 'gpt-4o', 
    model_provider => 'open_ai', 
    model_request_url =>'https://api.openai.com/v1/chat/completions', 
    model_type => 'generic', 
    model_auth_type => 'secret_manager', 
    model_auth_id => 'htn-openapi-key', 
    model_qualified_name => 'gpt-4o');
    
    And above, htn-openapi-key refers to a similarly declared secret as for HF below